index.html

```diff
@@ -499,18 +506,18 @@ <h2>Test Vectors</h2>
           "issuer_0": {
             "@context": [
               "https://www.w3.org/ns/did/v1",
-              "https://example.com/credentials/latest",
+              "https://w3c-ccg.github.io/lds-jws2020/contexts/lds-jws2020-v1.json",
```

Comment:
```suggestion
              "https://w3id.org/security/suites/jws-2020/v1",
```

Reply:
This was a mistake; should be `ed25519` instead of `jws`.

Reply:
feel free to resolve any suggestions that are wrong, you may find this helpful:

https://ns.did.ai/

don't use the context IRIs for this website, use the w3id.org ones, but we keep out own copy for testing and documentation purposes.

Reply:
Thanks @OR13.
I meant the change I made was wrong. Fixing now.